package.json

```diff
@@ -22,7 +22,7 @@
   "license": "MPL-2.0",
   "repository": "mozilla/notes",
   "scripts": {
```

Comment:
@udaraweerasinghege @pdehaan do we have a way to do something like:  `RTL=true npm start` or `LOCALE=he npm start` via the web-ext tool?